cmd/go-clang-generate/type.go

```diff
@@ -67,45 +71,58 @@ func getType(cType clang.Type) (Type, error) {
 	switch cType.Kind() {
 	case clang.TK_Char_S:
 		typ.CName = CSChar
-		typ.Name = string(GoInt8)
+		typ.Name = GoInt8
+		typ.Primitive = "schar"
```

Comment:
OK so "CName" is not really the C-name but the CGo name? I will correct that and rename some things.